### PR TITLE
fix: premiere bugs

### PR DESCRIPTION
**Pull request by domgan**
* renamed isShopItem to wider and more suited name: showPrice
* showing tooltip in shop even if item is not availible (no price)
* fixed sell component
* showing gold in sell component
* expanded "Poi" and changed order in skills component
* fixed attr sound, I will try to add better sounds to the storage

### CODE REVIEW

**Review by matfij — APPROVED** [review]
LGTM

File: src/views/shop/ShopBuyComponent.tsx [hunk 1]
```diff
@@ -23,8 +23,10 @@ const ShopBuyComponent = () => {
     const [loading, setLoading] = useState(false);
 
     useEffect(() => {
+        setLoading(true);
         void fetchShop();
-        void getGold();
+        void fetchGold();
+        setLoading(false);
```
Comment (matfij) [inline src/views/shop/ShopBuyComponent.tsx:29]: This will not be executed in correct order

Comment (domgan) [reply src/views/shop/ShopBuyComponent.tsx:29]: Order doesn't matter

Comment (matfij) [reply src/views/shop/ShopBuyComponent.tsx:29]: ???
Loading will be immediately set to false since void is just for making eslint happy

Comment (domgan) [reply src/views/shop/ShopBuyComponent.tsx]: yeah, I meant this is a bad change because it doesn't wait, good fix

**Review by matfij — APPROVED** [review]
LGTM

**Pull request merged**